docs/notebook-demos/reading-in-sentinel-data.ipynb

```diff
@@ -0,0 +1,225 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reading in Sentinel-2 Images\n",
+    "\n",
+    "Sentinel-2 is an observation mission developed by the European Space Agency to\n",
+    "monitor the surface of the Earth [official website](http://www.esa.int/Our_Activities/Observing_the_Earth/Copernicus/Sentinel-2).\n",
+    "Sets of images are taken of the surface where each image corresponds to a\n",
+    "specific wavelength. These images can provide useful data for a wide variety of\n",
+    "industries, however, the format they are stored can prove difficult to work\n",
+    "with. This being, `JPEG 2000` (file extension `.jp2`), an image compression\n",
+    "format for JPEGs that allows for improved quality and compression ratio.\n",
+    "\n",
+    "There are few programs that can work with `jp2`, which can make processing\n",
+    "large amounts of them difficult. Because of GeoPySpark, though, we can leverage\n",
+    "the tools available to us in Python so that can work with `jp2` and use them to\n",
+    "format the sentinel data so that it can be ingested and analyzed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting the Data\n",
+    "\n",
+    "Before we can start this tutorial, we will need to get the sentinel images.\n",
+    "All sentinel data can be found on Amazon's S3 service, and we will be downloading it\n",
+    "straight from there.\n",
+    "\n",
+    "The way the data is stored on S3 will not be discussed here, instead, a general\n",
```

Comment:
can say "the way its stored on AWS is discussed here" 

http://sentinel-pds.s3-website.eu-central-1.amazonaws.com/